homeassistant/components/light/zwave.py

```diff
@@ -107,8 +107,8 @@ def update_properties(self):
 
     def _value_changed(self, value):
         """Called when a value has changed on the network."""
-        if self._value.value_id == value.value_id or \
-           self._value.node == value.node:
+        if self._value.value_id != value.value_id or \
+           self._value.node != value.node:
```

Comment:
Wait, why should it not be equal? That means that the light will test for update anytime any value or node changes that is not the light ?